examples/resources/vmp_custom_rule_set/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    vmp = {
+      version = "0.4.0"
+      source  = "github.com/terraform-providers/vmp"
```

Comment:
For the final commit, let's refer to the provider path instead of the local install path. e.g.

`    ec = {
      version = "0.4.2"
      source  = "EdgeCast/ec"
    }`

Reply:
@wdawg33 you are correct, I pointed this out to @ssaluja4  during our meeting. I believe she has that change locally and it should be included in her final commit

Reply:
Updated. Thanks.